content/r/concepts/csv-files/csv-files.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'CSV Files'
+Description: 'Data within CSV files can be manipulated using built-in R functions.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'CSV'
+  - 'Files'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+Data within **[CSV](https://www.codecademy.com/resources/docs/general/csv) files** can be read and written using built-in functions.
+
+## Changing Directories
+
+The file being worked on should be located in the working directory. The current directory can be checked by printing the `getwd()` command:
```

Comment:
@Dusch4593 Perhaps we should change line 26 to "The current working directory can be set with the `setwd()` function, if needed:"  Or did you want to take out line 26 and the subsequent line altogether?

Reply:
@KTom101 looks like either could work. I think it would be good to change line 26 to the one suggested above since the example will follow right below